appveyor.yml

```diff
@@ -79,6 +81,12 @@
         appveyor_repo_tag: true       # deploy on tag push only
 
 -
+  branches:
+    except:
+      - release
+
+  skip_non_tags: false
```

Comment:
I added the `true` section in the tag config above, but maybe that's still not right